packages/experiments/src/components/M365Nav/M365NavToggler.tsx

```diff
@@ -0,0 +1,107 @@
+﻿/* tslint:disable */
+import { Icon } from 'office-ui-fabric-react/lib/components/Icon';
+import * as React from 'react';
+import { M365Nav } from './M365Nav';
+import {
```

Comment:
Can you remove all of the M365 everywhere? It is fine to even call it Nav and such.

Reply:
removed